configs/README.md

```diff
@@ -33,17 +33,20 @@ Please follow the outline structure and **table format** shown in [densenet/READ
 
 <div align="center">
 
-| Model        | Context  | Top-1 (%) | Top-5 (%) | Params (M) | Recipe                                                                                              | Download                                                                                           |
-|--------------|----------|-----------|-----------|------------|-----------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
-| densenet_121 | D910x8-G | 75.64     | 92.84     | 8.06       | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/densenet/densenet_121_ascend.yaml) | [weights](https://download.mindspore.cn/toolkits/mindcv/densenet/densenet121-120_5004_Ascend.ckpt) |
+| model       | top-1 (%) | top-5 (%) | params (M) | batch size | cards | ms/step | jit_level | recipe                                                                                              | download                                                                                                  |
+| ----------- | --------- | --------- | ---------- | ---------- | ----- | ------- | --------- | --------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------- |
+| densenet121 | 75.67     | 92.77     | 8.06       | 32         | 8     | 47,34   | O2        | [yaml](https://github.com/mindspore-lab/mindcv/blob/main/configs/densenet/densenet_121_ascend.yaml) | [weights](https://download-mindspore.osinfra.cn/toolkits/mindcv/densenet/densenet121-bf4ab27f-910v2.ckpt) |
 
 </div>
 
 Illustration:
 - Model: model name in lower case with _ seperator.
-- Context: Training context denoted as {device}x{pieces}-{MS mode}, where mindspore mode can be G - graph mode or F - pynative mode with ms function. For example, D910x8-G is for training on 8 pieces of Ascend 910 NPU using graph mode.
 - Top-1 and Top-5: Accuracy reported on the validatoin set of ImageNet-1K. Keep 2 digits after the decimal point.
 - Params (M): # of model parameters in millions (10^6). Keep **2 digits** after the decimal point
+- Batch Size: Training batch size
+- Cards: # of cards
+- Ms/step: Time used on training per step in ms
+- Jit_level: Jit level of mindspore context, which contains 3 levels: O0(kbk mode)/O1(dvm mode)/O2(ge mode)
```

Comment:
remove kbk/dvm/ge

Reply:
changed to: 
- Jit_level: Jit level of mindspore context, which contains 3 levels: O0/O1/O2